### PR TITLE
Functional CFR ToC tab

### PR DESCRIPTION
**Pull request by xtine**
Updated `DrawerEvents` so CFR ToC tab stays open while browsing CFR pages.
Related to eregs/notice-and-comment#117

### CODE REVIEW

**Comment by xtine** [comment]
@cmc333333: the `{{ cfr_change_toc }}` doesn't seem to be displaying in the `comment-review-chrome.html` template, am I missing something or does it need to be passed through the view to the review page?

**Comment by cmc333333** [comment]
It needs to be passed through, too. If possible, let's try to merge the relevant templates and views -- there's a lot of duplication.

File: regulations/static/regulations/js/source/views/main/preamble-view.js [hunk 1]
```diff
@@ -42,7 +42,14 @@ var PreambleView = ChildView.extend({
...
     CommentEvents.trigger('comment:readTabOpen');
-    DrawerEvents.trigger('pane:init', type === 'preamble' ? 'table-of-contents' : 'table-of-contents-secondary');
+
+    if (type === 'preamble') {
+      DrawerEvents.trigger('pane:init', 'table-of-contents');
+    }
+    else if (type === 'cfr') {
+      DrawerEvents.trigger('pane:init', 'table-of-contents-secondary');
+      DrawerEvents.trigger('pane:change', 'table-of-contents-secondary');
+    }
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/preamble-view.js]: @jmcarp heads up; related to what you're looking at.

**Comment by cmc333333** [comment]
@xtine to confirm, this isn't ready for review yet, right?

**Comment by xtine** [comment]
Not yet, just wanted to get your advice on the template situation.

**Pull request merged**